config/webpack.config.prod.js

```diff
@@ -244,7 +244,7 @@ module.exports = {
     // Generates an `index.html` file with the <script> injected.
     new HtmlWebpackPlugin({
       inject: true,
-      template: paths.appHtml,
+      template: '!!prerender-loader?string!' + paths.appHtml,
```

Comment:
Thanks for the PR! This currently appears to be failing locally on child compilation:

```
Creating an optimized production build...
Failed to compile.

  Error: Child compilation failed:
  Module build failed: Error: Child compilation failed:

  - prerender-loader.js:23
    [react-shrine]/[prerender-loader]/dist/prerender-loader.js:23:31

  - Compiler.js:514 applyPluginsAsync.err
    [react-shrine]/[webpack]/lib/Compiler.js:514:14

  - Tapable.js:202 next
    [react-shrine]/[tapable]/lib/Tapable.js:202:11

  - CachePlugin.js:78 Compiler.<anonymous>
    [react-shrine]/[webpack]/lib/CachePlugin.js:78:5

  - Tapable.js:206 Compiler.applyPluginsAsyncSeries
    [react-shrine]/[tapable]/lib/Tapable.js:206:13

  - Compiler.js:511 compilation.seal.err
    [react-shrine]/[webpack]/lib/Compiler.js:511:11

  - Tapable.js:195 Compilation.applyPluginsAsyncSeries
    [react-shrine]/[tapable]/lib/Tapable.js:195:46

  - Compilation.js:680 self.applyPluginsAsync.err
    [react-shrine]/[webpack]/lib/Compilation.js:680:19

  - Tapable.js:195 Compilation.applyPluginsAsyncSeries
    [react-shrine]/[tapable]/lib/Tapable.js:195:46

  - Compilation.js:671 self.applyPluginsAsync.err
    [react-shrine]/[webpack]/lib/Compilation.js:671:11

  - Tapable.js:202 next
    [react-shrine]/[tapable]/lib/Tapable.js:202:11

  - index.js:244 Compilation.<anonymous>
    [react-shrine]/[uglifyjs-webpack-plugin]/dist/index.js:244:6

  - Tapable.js:206 Compilation.applyPluginsAsyncSeries
    [react-shrine]/[tapable]/lib/Tapable.js:206:13

  - Compilation.js:666 self.applyPluginsAsync.err
    [react-shrine]/[webpack]/lib/Compilation.js:666:10

  - Tapable.js:195 Compilation.applyPluginsAsyncSeries
    [react-shrine]/[tapable]/lib/Tapable.js:195:46


  - Error: Child compilation failed:

  - compiler.js:76
    [react-shrine]/[html-webpack-plugin]/lib/compiler.js:76:16

  - Compiler.js:304 compile
    [react-shrine]/[webpack]/lib/Compiler.js:304:11

  - Compiler.js:514 applyPluginsAsync.err
    [react-shrine]/[webpack]/lib/Compiler.js:514:14

  - Tapable.js:202 next
    [react-shrine]/[tapable]/lib/Tapable.js:202:11

  - CachePlugin.js:78 Compiler.<anonymous>
    [react-shrine]/[webpack]/lib/CachePlugin.js:78:5

  - Tapable.js:206 Compiler.applyPluginsAsyncSeries
    [react-shrine]/[tapable]/lib/Tapable.js:206:13

  - Compiler.js:511 compilation.seal.err
    [react-shrine]/[webpack]/lib/Compiler.js:511:11

  - Tapable.js:195 Compilation.applyPluginsAsyncSeries
    [react-shrine]/[tapable]/lib/Tapable.js:195:46

  - Compilation.js:680 self.applyPluginsAsync.err
    [react-shrine]/[webpack]/lib/Compilation.js:680:19

  - Tapable.js:195 Compilation.applyPluginsAsyncSeries
    [react-shrine]/[tapable]/lib/Tapable.js:195:46

  - Compilation.js:671 self.applyPluginsAsync.err
    [react-shrine]/[webpack]/lib/Compilation.js:671:11

  - Tapable.js:202 next
    [react-shrine]/[tapable]/lib/Tapable.js:202:11

  - index.js:244 Compilation.<anonymous>
    [react-shrine]/[uglifyjs-webpack-plugin]/dist/index.js:244:6

  - Tapable.js:206 Compilation.applyPluginsAsyncSeries
    [react-shrine]/[tapable]/lib/Tapable.js:206:13

  - Compilation.js:666 self.applyPluginsAsync.err
    [react-shrine]/[webpack]/lib/Compilation.js:666:10

  - Tapable.js:195 Compilation.applyPluginsAsyncSeries
    [react-shrine]/[tapable]/lib/Tapable.js:195:46
```

Reply:
Taking a look. Usually it's an eagerly accessed global or incompatible plugin/output.globalObject.